.prettierrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
```

Comment:
I think the changes to Prettier are not necessary. Moreover, you seem to be using entirely different settings, which results in reformatting the entire codebase. I'd ask you to revert that if you're okay.

Reply:
I'm sorry, that was accidental. I did not mean to change that. I'll revert it.